measures/SimulationOutputReport/measure.rb

```diff
@@ -130,8 +130,13 @@ def run(runner, user_arguments)
     # Load buildstock_file
     resources_dir = File.absolute_path(File.join(File.dirname(__FILE__), "..", "..", "lib", "resources")) # Should have been uploaded per 'Other Library Files' in analysis spreadsheet
     buildstock_file = File.join(resources_dir, "buildstock.rb")
-    require File.join(File.dirname(buildstock_file), File.basename(buildstock_file, File.extname(buildstock_file)))
-
+    if File.exists? buildstock_file
```

Comment:
Can you tell me the scenario where this is useful?

Reply:
When running `SimulationOutputReport` in an osw locally, I receive an error looking for `/resstock/lib/resources/buildstock.rb`, which doesn't exist. This change points to the `buildstock.rb` in `resources` instead (https://github.com/NREL/resstock/blob/develop/resources/buildstock.rb)

Reply:
This is probably fine. But I think you should put comments in there explaining why.